src/components/Footer/Footer.js

```diff
@@ -140,19 +141,8 @@ function Footer() {
             <div className="col-md-4">
```

Comment:
Please add test coverage for this up at as well.

Reply:
Any of the form tests should be similar to what is needed to test this form in the footer.

Reply:
Haha, my bad. I thought it was strictly referring to the div with updated class. Thank you!